services/director-v2/tests/conftest.py

```diff
@@ -18,6 +21,9 @@
 from simcore_service_director_v2.core.settings import AppSettings, BootModeEnum
 from starlette.testclient import TestClient
 
+nest_asyncio.apply()
```

Comment:
what is this for?

Reply:
Because I am using async def tests, I needed to apply this operator in order to not refactor existing tests. I spent quite some time to figure out that this was the best solution.